docs/SMCWG-charter.md

```diff
@@ -287,13 +287,12 @@ three (3).
   to the Chair herein MAY be performed by the Vice Chair. For example, the Vice
   Chair MAY preside at SMCWG Meetings and Teleconferences in the Chair’s
   absence.
-- SMCWG-created Subcommittees MAY be approved either:
-
-  1. by formal ballot as described in [6.1](#6.1-voting-structure), or
-  2. by simple majority vote of those members present at a regularly
-     scheduled SMCWG Meeting or Teleconference provided that the proposal is
-     mentioned in an agenda circulated on the SMCWG Mail List at least
-     forty-eight (48) hours prior to the SMCWG Meeting or Teleconference.
+- The SMCWG may establish subcommittees, by formal ballot, to address any of the
```

Comment:
Having formal ballots for subcommittees doesn't really slow things down that much, and provides more clarity about what they are for and who is for them.